src/server/migrations/20200317164418_add_batches_table.js

```diff
@@ -0,0 +1,31 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('batches', function(table) {
+    table
+      .increments('id')
+      .primary()
+      .notNullable();
+    table.integer('fk_crop_id').notNullable();
+    table.string('customer_name').notNullable();
+    table
+      .timestamp('seeding_date')
+      .defaultTo(knex.fn.now())
+      .notNullable();
+    table
+      .timestamp('updated_at')
+      .defaultTo(knex.fn.now())
+      .notNullable();
+    table
+      .timestamp('created_at')
+      .defaultTo(knex.fn.now())
+      .notNullable();
+    table.timestamp('deleted_at');
+    table
```

Comment:
Can you please follow the structure for all tables: like this "created_at, updated_at and deleted_at".